### ISSUE TITLE
Feature request: Upload on save without confirmation for files in selected folders to selected remote paths

### ISSUE THREAD

**Issue by ArvidBaa** [humy2833/FTP-Simple#57]
I would like to be able to save a local file and upload it to the ftp server automatically, without having to confirm the remote path. I have set up the "project" setting, so that I automatically get the correct path in the dropdown folder, and I have also changed the key-binding for saving to ftp as cmd-S instead of the default cmd-shift-S. This means that I can save a local file as usual and upload it to the server, except that I need to hit enter to confirm the remote path for upload. I would like to take away this confirmation step, so that I can work locally as if I was working remotely.

As it is now, however, any file opened that are being saved will prompt the save to remote. That is also annoying, since I know that I will always only work in directories that have been set up in the settings files as "project" directories and connected to a corresponding remote path.

The solution I'm asking for therefore needs to include some setting to make sure that not any random file that is being saved automatically gets uploaded to some folder on the ftp server.

So, put simply, I want to be able to upload on save without confirmation for files in selected folders (given by the setting "project" or by some other setting). Could that be possible?

**Comment by humy2833** [comment]
Added to version 0.4.23.
You can set it as follows. ("save" option)
```json
"project": {
			"D:/dev/lib": {"path":"/html/lib", "save":true}
		}
```

Thanks.
